hearpreprocess/pipeline.py

```diff
@@ -715,7 +776,8 @@ def requires(self):
 
     def run(self):
         audiofiles = set(self.requires()["corpus"].workdir.glob("*.wav"))
-        for audiofile in audiofiles:
+        split_count = {split: 0 for split in SPLITS}
+        for audiofile in tqdm(audiofiles):
```

Comment:
We don't need tqdm for things that are nearly instantaneous, it will just clutter the luigi output

Reply:
This takes some time for large corpora; for the full set basically

Reply:
okay